compiler/wasm/Cargo.toml

```diff
@@ -32,3 +32,8 @@ getrandom = { workspace = true, features = ["js"] }
 
 [build-dependencies]
 build-data.workspace = true
+
+# `wasm-opt` is on by default in for the release profile, but it can be
+# disabled by setting it to `false`
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
```

Comment:
Why do we no longer run wasm-opt?

Reply:
Missed this from dev, restored. Unfortunately it makes building the wasm super slow, will look into conditional optimizations depending on env.